simconnect-sdk/src/domain/client_event.rs

```diff
@@ -126,6 +162,40 @@ pub enum ClientEvent {
     },
     /// Toggles parking brake on/off.
     ParkingBrakes,
+    // Comm Radios
+    Com1RadioStbySet {
```

Comment:
It would be good to add the documentation that's found at https://docs.flightsimulator.com/html/Programming_Tools/Event_IDs/Aircraft_Radio_Navigation_Events.htm#NAV1_STBY_SET_HZ to the events and their inner data.